common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java

```diff
@@ -396,6 +404,35 @@ public void onInfoMessage(ChatMessageReceivedEvent e) {
                     return;
                 }
             }
+
+            if (potion != FilterType.KEEP) {
+                Matcher maxPotionsMatcher = MAX_POTIONS_ALLOWED_PATTERN.matcher(msg);
+                if (maxPotionsMatcher.matches()) {
+                    e.setCanceled(true);
+                    if (potion == FilterType.HIDE) {
+                        return;
+                    }
+
+                    NotificationManager.queueMessage(
+                            new TextComponent("At Potion Charge Limit!").withStyle(ChatFormatting.DARK_RED));
```

Comment:
why is the message capitalized

Reply:
in my head i thought it would play better with the already existing "+3 Potion Charges" message but that can be changed

Reply:
If you look at the other message it is not consisgent

Reply:
i see your point

Reply:
@kristofbolyai I think at some point (maybe when doing the i18n?) we need to have a look at all the notification messages as a whole to make sure coloring, capitalization and punctuation is consistent. Right now it feels a bit random if a message ends with a period, an exclamation mark or nothing at all (which I think makes the most sense, for this short phrases). 

Maybe we can classify them as "info", "warning" or "error" and use colors based on that. "No totems nearby" would be an error, "+3 potion charges" and info and "At potion charge limit" a warning. Or something like that.

@san7890 Not saying you need to do any of this in this PR, just thinking out loud.

Reply:
> @kristofbolyai I think at some point (maybe when doing the i18n?) we need to have a look at all the notification messages as a whole to make sure coloring, capitalization and punctuation is consistent. Right now it feels a bit random if a message ends with a period, an exclamation mark or nothing at all (which I think makes the most sense, for this short phrases). 
> 
> 
> 
> Maybe we can classify them as "info", "warning" or "error" and use colors based on that. "No totems nearby" would be an error, "+3 potion charges" and info and "At potion charge limit" a warning. Or something like that.
> 
> 
> 
> @san7890 Not saying you need to do any of this in this PR, just thinking out loud.

Perhaps modify the i18n issue to include this?

Reply:
or make a new..